packages/spec/src/locales.ts

```diff
@@ -262,6 +263,6 @@ export const resolveOverridenLocale = (locale: string, delimiter?: "-" | "_" | n
   return locale.replace(currentDelimiter, delimiter);
 };
 
-export function normalizeLocale(locale: string) {
+export function normalizeLocale(locale: string): string {
```

Comment:
This is not necessary since Typescript automatically infers simple types like this. But it also does not hurt to be explicit since the method is exported from the lib 🙃